common.eai

```diff
@@ -5140,15 +5135,14 @@ endfunction
 //============================================================================
 function GetHeroStrength takes unit u returns integer
   local integer lvl = GetHeroLevel(u)
-  if not IsUnitInGroup(u, unit_healing) then
+  if GetOwningPlayer(u) != ai_player or not IsUnitInGroup(u, unit_healing) then
```

Comment:
Not necessary to limit this to just this player, would result in problems if we did use it for other players heroes and could have its uses.

Reply:
I thought `like unit_healing this group` only add AI own unit , so restriction was added to avoid calling `IsUnitInGroup` too much
Because I superstitiously believe that calling `IsUnitInGroup` too much can also cause lag .......